controllers/accountControllers/userAccountController.js

```diff
@@ -0,0 +1,132 @@
+const {Account} = require('../../models/userAccountModel');
+const {User} = require('../../models/userModel');
+const {validateAccount} = require('../../validation/validationAccount');
+
+
+const createAccount =  async (req, res, next) => {
+    const { error } = await validateAccount.validate(req.body);
+    if(error){
+        return next (new Error(error));
+    }
+    const {
+        accountName, 
+        accountCurrency, 
+        accountAmount, 
+        accountDescription, 
+        accountStat
+    }   = req.body;
+    
+    if (
+        accountName && 
```

Comment:
Is this condition needed? `mongoose` validation should handle missing properties

Reply:
I removed that. I think I put double validations